go/vt/mysqlctl/proto/schema.go

```diff
@@ -17,10 +16,13 @@ import (
 )
 
 const (
+	// TABLE_BASE_TABLE indicates the table type is a base table.
 	TABLE_BASE_TABLE = "BASE TABLE"
```

Comment:
Will do another PR for this since TABLE_BASE_TABLE is referenced by many other places (including internal one)